go/vt/vttablet/tabletmanager/vreplication/vplayer.go

```diff
@@ -198,6 +203,22 @@ func (vp *vplayer) play(ctx context.Context) error {
 	}
 }
 
+func (vp *vplayer) buildTableKeys() (map[string][]string, error) {
+	schema, err := vp.mysqld.GetSchema(vp.dbClient.DBName(), []string{"/.*/"}, nil, false)
+	if err != nil {
+		return nil, err
+	}
+	tableKeys := make(map[string][]string)
+	for _, td := range schema.TableDefinitions {
+		if len(td.PrimaryKeyColumns) != 0 {
+			tableKeys[td.Name] = td.PrimaryKeyColumns
+		} else {
+			tableKeys[td.Name] = td.Columns
```

Comment:
Is there a test covering this case? Where the table does not have PK's and it defaults to all columns? I did not see that, but it might have been covered in previous PR.

Reply:
This should be covered by: https://github.com/vitessio/vitess/blob/3d917789201468b342a3af2f96970adfb6afe105/go/vt/vttablet/tabletmanager/vreplication/vplayer_test.go#L48